transport/server.cc

```diff
@@ -1218,6 +1218,10 @@ std::unique_ptr<cql_server::response>
 cql_server::connection::make_result(int16_t stream, shared_ptr<messages::result_message> msg, const tracing::trace_state_ptr& tr_state, bool skip_metadata)
 {
     auto response = std::make_unique<cql_server::response>(stream, cql_binary_opcode::RESULT, tr_state);
+    if (__builtin_expect(!msg->warnings().empty() && _version > 3, false)) {
+        response->set_frame_flag(cql_frame_flags::warning);
+        response->write_string_list(msg->warnings());
```

Comment:
Nit: warnings could probably be moved from

Reply:
write_string_list() should be changed to accept its input by const reference, since it can't take advantage of a move anyway.